frontend/src/app/core/setup/globals/onboarding/tours/homescreen_tour.ts

```diff
@@ -15,25 +10,10 @@ export function homescreenOnboardingTourSteps():OnboardingStep[] {
       bottom: 7,
     },
     {
-      description: I18n.t('js.onboarding.steps.project_selection'),
-      selector: '.widget-box.welcome',
-      event: 'custom',
-      showSkip: false,
       containerClass: '-dark -hidden-arrow',
-      clickable: true,
       onBeforeStart() {
-        // Handle the correct project selection and redirection
-        // This will be removed once the project selection is implemented
-        jQuery(`.widget-box.welcome a:contains(${scrumDemoProjectName})`).click(function (this:HTMLAnchorElement) {
-          window.onboardingTourInstance.trigger('next');
-          window.location.href = `${this.href}/?start_scrum_onboarding_tour=true`;
-        });
-        jQuery(`.widget-box.welcome a:contains(${demoProjectName})`).click(function (this:HTMLAnchorElement) {
-          window.onboardingTourInstance.trigger('next');
-          window.location.href = `${this.href}/work_packages/?start_onboarding_tour=true`;
-        });
-        // Disable clicks on other links
-        jQuery('.widget-box.welcome a').addClass('-disabled').bind('click', preventClickHandler);
+        const href = window.location.href.slice(0, -1);
```

Comment:
What is the reason for slicing here if we start from position 0?

Reply:
I wanted to remove '?' from the end of the href (http://localhost:3000/?)

Reply:
I see. You can also use `window.location.origin`, which should be less error prone.